### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by hung-eoh**
Change library URL
It is currently listed in the Library Manager as
https://github.com/eoh-jsc/mvp-lib
It is now:
https://github.com/eoh-jsc/era-lib

### CODE REVIEW

**Comment by per1234** [comment]
Hi @hung-eoh. I see that the bot detected that the library name was changed from "EoH Platform" to "ERa":

https://github.com/eoh-jsc/era-lib/commit/1932785cefb1647c55e1d46c4d7e1eef70f61ee5#diff-f267f9ae2f5d5d56a1dbb305e95d40d4408d9ee7f71357f1f67fb18c29b7f082

Because it is the sole unique identifier, libraries are locked to the name they had at the time of the original submission. The registered name can be changed, but this is only done by the explicit request of the library maintainer.

Please tell me whether you would also like the registered name to be updated in addition to the URL.

**Comment by per1234** [comment]
Aha! I just noticed https://github.com/arduino/library-registry/issues/2025. That will serve as the name change request. I'll prepare the registry side changes right now.

**Comment by hung-eoh** [comment]
> Aha! I just noticed #2025. That will serve as the name change request. I'll prepare the registry side changes right now.

Thank you so much

**Comment by per1234** [comment]
Hello again @hung-eoh. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/2026#issuecomment-1308493649

**Pull request merged**
